src/main/scala/cromiam/server/config/CromIamServerConfig.scala

```diff
@@ -10,15 +10,17 @@ import scala.collection.JavaConverters._
 import scala.util.{Failure, Success, Try}
 import cromiam.server.config.CromIamServerConfig._
 
-final case class CromIamServerConfig(cromIamConfig: CromIamConfig, cromwellConfig: ServiceConfig, samConfig: ServiceConfig)
+final case class CromIamServerConfig(cromIamConfig: CromIamConfig, cromwellConfig: ServiceConfig,
```

Comment:
It's been a long time since I used a VIC-20. 

The reason why I don't like a hard cap is because I disagree with having a hard cap. I like keeping things to around 120-130 but sometimes it's just stupid to force a line wrap at that point (e.g. it's blatantly obvious what comes after anyways) and I haven't seen a formatter yet that can discern the difference.